src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```

Comment:
you can set `margin-right: 4px;` here, and use `:not` with `:last-child` to prevent setting classes for the last element

Reply:
got you, thanks!